services/workflows-service/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "workflow-service",
   "scripts": {
     "spellcheck": "cspell \"*\"",
-    "setup": "npm run docker:db:down && npm run docker:db && npm run db:reset && npm run seed",
+    "setup": "npm run docker:db:down && npm run docker:db && sleep 10 && npm run db:reset && npm run seed",
```

Comment:
Hey @riturajgc , thanks for the contribution! Ideally the solution would wait for the database to be ready by using something like a lifecycle hook or similar over a sleep. Is this something you're willing to explore?

Reply:
I can see we use a package called `wait-on`, could you check if by waiting on the db port is solves the issue? 🙏🏼